linchpin/provision/roles/aws/tasks/teardown_aws_ec2.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: "Debug"
+  debug:
+    msg: "{{ tp_out }}"
+
```

Comment:
This needs to be commented out at a minimum because it will print account information.